CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
Seems like there's an open issue for that: https://github.com/twolfson/restructuredtext-lint/issues/8